### PR TITLE
v2podman port

### PR DESCRIPTION
**Pull request by baude**
add port command to podman.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: pkg/domain/entities/containers.go [hunk 1]
```diff
@@ -341,3 +342,16 @@ type ContainerPruneReport struct {
 	ID  map[string]int64
 	Err map[string]error
 }
+
+// ContianerPortOptions describes the options to obtain
```
Comment (TomSweeneyRedHat) [inline pkg/domain/entities/containers.go:346]: typo  ContianerPortOptions -> ContainerPortOptions

File: pkg/domain/entities/engine_container.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ type ContainerEngine interface {
 	ContainerLogs(ctx context.Context, containers []string, options ContainerLogsOptions) error
 	ContainerMount(ctx context.Context, nameOrIds []string, options ContainerMountOptions) ([]*ContainerMountReport, error)
 	ContainerPause(ctx context.Context, namesOrIds []string, options PauseUnPauseOptions) ([]*PauseUnpauseReport, error)
+	ContainerPort(ctx context.Context, options ContainerPortOptions) ([]*ContainerPortReport, error)
```
Comment (jwhonce) [inline pkg/domain/entities/engine_container.go:31]:
```suggestion
	ContainerPort(ctx context.Context, nameOrId string, options ContainerPortOptions) ([]*ContainerPortReport, error)
```
Would be more consistent with other methods.

File: pkg/domain/entities/containers.go [hunk 3]
```diff
@@ -341,3 +342,16 @@ type ContainerPruneReport struct {
...
+// port information on containers
+type ContainerPortOptions struct {
+	All       bool
+	Container string
```
Comment (jwhonce) [inline pkg/domain/entities/containers.go:350]:
```suggestion
```
nameOrId should be a parameter not an option

**Comment by baude** [comment]
fixed and ready

**Pull request merged**